src/AAL.xml

```diff
@@ -0,0 +1,83 @@
+        may help justify the thousands of dollars in otherwise billable time invested in writing this and other
+        freely available, open-source software. 
+      </p>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>Redistributions of source code, in whole or part and with or without modification (the "Code"),
+             must prominently display this GPG-signed text in verifiable form.</p>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>Redistributions of the Code in binary form must be accompanied by this GPG-signed text in any
+             documentation and, each time the resulting executable program or a program dependent thereon
+             is launched, a prominent display (e.g., splash screen or banner text) of the Author's
+             attribution information, which includes:</p>
+        </li>
+        <list>
+          <li>
+            <b>(a)</b>
+            <p>
+              <alt name="author" match=".+">Name</alt> ("AUTHOR"), 
+            </p>
+          </li>
+          <li>
+            <b>(b)</b>
+            <p>
+              <alt name="profIdent" match=".+">Professional identification</alt> ("PROFESSIONAL IDENTIFICATION"), and 
+            </p>
+          </li>
+          <li>
+            <b>(c)</b>
+            <p>
+              <alt name="url" match=".+">URL</alt> ("URL"). 
+            </p>
+          </li>
+        </list>
+        <li>
+          <b>3.</b>
+          <p>Neither the name nor any trademark of the Author may be used to endorse or promote products
+             derived from this software without specific prior written permission.</p>
+        </li>
+        <li>
+          <b>4.</b>
+          <p>Users are entirely responsible, to the exclusion of the Author and any other persons, for
+             compliance with (1) regulations set by owners or administrators of employed equipment, (2)
+             licensing terms of any other software, and (3) local regulations regarding use, including
+             those regarding import, export, and use of encryption software.</p>
+        </li>
+      </list>
+      <p>THIS FREE SOFTWARE IS PROVIDED BY THE AUTHOR "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING,
+         BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+         DISCLAIMED. IN NO EVENT SHALL THE AUTHOR OR ANY CONTRIBUTOR BE LIABLE FOR ANY DIRECT, INDIRECT,
+         INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, EFFECTS OF
+         UNAUTHORIZED OR MALICIOUS NETWORK ACCESS; PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+         DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+         CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+         USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.</p>
```

Comment:
there is some additional text on OSI site - should this have been here too and then marked as optional? http://spdx.org/licenses/AAL.html